tools/WinObjC.Tools/vsimporter-templates/WinStore10-App/Package-native.appxmanifest

```diff
@@ -43,6 +43,8 @@
     </Application>
   </Applications>
 
+  $extensions$
```

Comment:
>$extensions$ [](start = 2, length = 12)

Use <Extensions VSImporterLabel="Extensions" /> in combination with pugixml instead